src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -50,6 +61,11 @@ def setup(self, builder: Builder):
                 "address_id": data_values.GQ_HOUSING_TYPE_MAP.keys(),
                 "housing_type": data_values.GQ_HOUSING_TYPE_MAP.values(),
                 "po_box": data_values.NO_PO_BOX,
+                # Add dummy state_id and puma during setup b/c we do not have
```

Comment:
We can actually modify vectorized choice to be able to function without a randomness stream and still preserve CRN. Let's talk tomorrow about how to do this.